source/connect/favorite-connections/import-export-cli/import.txt

```diff
@@ -12,4 +12,153 @@ Import Favorite Connections with the CLI
    :depth: 2
    :class: singlecol
 
-TODO: Write
+You can use the |compass-short| :abbr:`CLI (Command-Line Interface)` to
+import favorite connections. This lets you use saved connections from
+other workspaces or connections used by other team members.
+
+Prerequisite
+------------
+
+To import favorite connections, you must first :ref:`export the
+connections <compass-export-connections-cli>`.
+
+Import Unencrypted Favorite Connections
+---------------------------------------
+
+Use this procedure to import a list of favorite connections that is not
+encrypted with a passphrase.
+
+Procedure
+~~~~~~~~~
+
+To import unencrypted favorite connections with the |compass-short|
```

Comment:
Is separating out `Procedure` a common docs practice? If so ignore.
I was thinking it might read a little smoother and more concisely if we instead write it in one go, something like:
```suggestion
To import favorite connections that are not encrypted with a passphrase 
using the |compass-short| :abbr:`CLI (Command-Line Interface)`, specify:
```
Similar question/thought below for the `Procedure` on Encrypted Favorite Connections

Reply:
Thanks for calling this out. We've been shifting to adopting more of a consistent page structure for task pages so users know what to expect when they arrive to a docs page. Part of that structure includes putting steps / procedures into a dedicated section. I think this can also help users who use the table of contents on the right-side to navigate a page, since they can jump right to the steps.